node/cmd/guardiand/adminserver.go

```diff
@@ -466,3 +466,22 @@ func (s *nodePrivilegedService) ChainGovernorReleasePendingVAA(ctx context.Conte
 		Response: resp,
 	}, nil
 }
+
+func (s *nodePrivilegedService) ChainGovernorResetReleaseTimer(ctx context.Context, req *nodev1.ChainGovernorResetReleaseTimerRequest) (*nodev1.ChainGovernorResetReleaseTimerResponse, error) {
+	if s.governor == nil {
+		return nil, fmt.Errorf("chain governor is not enabled")
+	}
+
+	if len(req.VaaId) == 0 {
+		return nil, fmt.Errorf("the VAA id must be specified as \"chainId/emitterAddress/seqNum\"")
```

Comment:
Should this validate full request is valid rather than check it's simply not an empty string?

Reply:
this is just an admin command, nvmd.